### PR TITLE
More Scratch stuff now with sensors!

### PR DESCRIPTION
**Pull request by CleoQc**
GPG3 support is complete.
Now I have to deal with sensors

### CODE REVIEW

**Comment by CleoQc** [comment]
Currently supports:
1. Distance Sensor
2. Ultrasonic Sensor
3. Light Sensor
4. Buzzer
5. LED
6. Line Follower (but I still have an issue with it, so it's not really tested)

**Comment by CleoQc** [comment]
Not tested but simply ported from BrickPi3Scratch:
Camera support and Speak support (text to speed)

**Comment by CleoQc** [comment]
Servos are now supported, both in easygopigo3 and in Scratch

File: Software/gopigo3_power.py [hunk 1]
```diff
@@ -11,6 +11,10 @@
 import RPi.GPIO as GPIO
 import time
 import os
+import atexit
```
Comment (DexterInd) [inline Software/gopigo3_power.py]: @CleoQc can we chuck this?

File: Software/gopigo3_power.py [hunk 2]
```diff
@@ -11,6 +11,10 @@
...
+
+atexit.register(GPIO.cleanup)
```
Comment (DexterInd) [inline Software/gopigo3_power.py]: @CleoQc can we chuck this?

**Pull request merged**